docs/guided-tour/getting-started.md

```diff
@@ -120,7 +120,7 @@ block runs when it completes.
 **Blocks** can contain a single job or many parallel jobs. For example, your
 `Tests` block contains two parallel jobs.
 
-!!! Info: "Running parallel jobs simultaneously is possible under open source, free trial, or paid [plans][plans]."
+❗Info: "Running parallel jobs simultaneously is possible under open source, free trial, or paid [plans][plans]."
```

Comment:
are you sure about this formatting change? it's special markup for an info box, it's not supposed to be an exclamation mark